doc/source/overview.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+
+## Raster Data
```

Comment:
The term "raster" I believe refers to an image that is rendered by scanning as in a CRT. Maybe it's just me, but I dislike this term. "Grid" or "gridded" seems preferable to me. Can we change that?

Reply:
I was trying to get at the netCDF data model of a cube or hypercube without getting into the details of the netCDF format, which, despite being not directly accessible to the API user, do affect how data is organized. I don't like "gridded" because it implies two-dimensionality. Maybe title this section, uh, Data Matrix?

Reply:
Yeah, I'd really prefer that. But if you think it too strained, it's not that big a deal.